package.json

```
@@ -189,5 +189,8 @@
   "resolutions": {
     "**/react-is": "^18.2.0",
     "**/@types/node": "^18.18.6"
+  },
+  "engines": {
+    "node": ">= 18"
```

Nitpick: The space is not needed and is used when there is a need to define a complex range. 😉 
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines
```suggestion
    "node": ">=18"
```